cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -133,11 +216,21 @@ int32_t cros_gralloc_driver::allocate(const struct cros_gralloc_buffer_descripto
 	uint64_t use_flags;
 	int32_t reserved_region_fd;
 	char *name;
+	bool from_kms;
```

Comment:
from_kms is uninitialized if "descriptor->use_flags & BO_USE_SCANOUT" is false?

Reply:
ok, it should be init as false then set to the hnd->from_kms.